src/supermarket/app/controllers/api/v1/cookbook_versions_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::CookbookVersionsController < Api::V1Controller
   before_action :check_cookbook_name_present, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation]
-  before_action :check_authorization, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation]
-  before_action :find_cookbook_version, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation]
+  before_action :check_authorization, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation, :license_evaluation]
+  before_action :find_cookbook_version, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation, :license_evaluation]
```

Comment:
This controller is starting to look ripe for an overhaul. All of the `_evaluation` endpoints are about creating `MetricResult` resources on a `CookbookVersion`. Starting to smell something like an `Api::V1::MetricResultsController`. I'd really like to consider a refactor ...or renovation... of this controller in this go-around.